app/assets/javascripts/app/views/notifications_view.js

```diff
@@ -28,23 +28,23 @@ app.views.Notifications = Backbone.View.extend({
 
   setUnreadStatus: function(guid, state){
     $.ajax({
-      url: "/notifications/" + guid,
+      url: '/notifications/' + guid,
       data: { set_unread: state },
-      type: "PUT",
+      type: 'PUT',
       context: this,
       success: this.clickSuccess
     });
   },
 
   clickSuccess: function(data) {
-    var type = $('.stream_element[data-guid=' + data["guid"] + ']').data('type');
-    this.updateView(data["guid"], type, data["unread"]);
+    var type = $('.stream_element[data-guid=' + data['guid'] + ']').data('type');
+    this.updateView(data['guid'], type, data['unread']);
```

Comment:
['guid'] is better written in dot notation.<br>['unread'] is better written in dot notation.